insights/tests/parsers/test_sys_fs_cgroup_memory_tasks_number.py

```diff
@@ -0,0 +1,12 @@
+from insights.parsers.sys_fs_cgroup_memory_tasks_number import SysFsCgroupMemoryTasksNumber
+from insights.tests import context_wrap
+
+
+Tasks_Number = """
+260
+""".strip()
+
+
+def test_sys_fs_cgroup_memory_tasks_number():
+    sys_fs_cgroup_memory_tasks_number_content = SysFsCgroupMemoryTasksNumber(context_wrap(Tasks_Number))
```

Comment:
since 100% coverage is also required for datasource tests, would you please add some abnormal cases to let the coverage reach 100%?